pkgs/os-specific/linux/nvidia-x11/builder.sh

```diff
@@ -191,8 +191,8 @@ installPhase() {
                     --set-rpath $out/lib:$libPath $bin/bin/$i
             fi
         done
-        # FIXME: needs PATH and other fixes
-        # install -Dm755 nvidia-bug-report.sh $bin/bin/nvidia-bug-report.sh
+        substituteInPlace nvidia-bug-report.sh --replace /bin/grep grep --replace /bin/ls ls
```

Comment:
There was a similar problem with one of the /bin/ls invocations, masked by the script's error handling. Fixed.